requirements.txt

```diff
@@ -1,9 +1,10 @@
 pyhive[hive_pure_sasl]~=0.7.0
 requests>=2.28.1
 
-pyodbc~=5.0.1 --no-binary pyodbc
+pyodbc~=4.0.39 --no-binary pyodbc
```

Comment:
downgrading pyodbc?

Reply:
It was inadvertently upgraded in this file due to `dependabot`, but `setup.py` was never updated. So we were testing the wrong version.

https://github.com/dbt-labs/dbt-spark/blob/94af50e683c7c4466708e40e29ce38a51ee102b8/setup.py#L47